pkg/mock/scheduledevents/scheduledevents.go

```diff
@@ -71,7 +71,7 @@ func getMetadata() t.Event {
 	eventResp := t.Event{
 		Code:              se.EventCode,
 		Description:       descriptionPrefix + se.EventCode,
-		EventID:           md.EventID,
```

Comment:
I think this change is enough to ensure parity with IMDS. 

We probably don't need the changes in */types.go files above -- we will have to deal with go-lint failures otherwise as golang likes 'ID'

Reply:
Ah, didn't know that. Would go-lint errors already have surfaced via Travis or are you saying if/when we run it in the future?

Reply:
good point! This gets json marshaled somewhere, right?  Can we just add json tags (in addition to the mapstructure tag) in types.go for marshaling so that the linter doesn't yell?

Reply:
I see go-lint complain in my IDE. However, go-report-card test didnt catch this. So, not very sure. 
But, for what we are trying to accomplish here, dont think the field names in the code matter anyway. 

I cant find event-id static path here - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-categories.html 
Am I missing something?

Reply:
EventId isn't its own path, it's an attribute of the returned blob from 'events/maintenance/scheduled' path

EventId in an example scheduled event in the docs: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/monitoring-instances-status-check_sched.html#viewing_scheduled_events

Reply:
> good point! This gets json marshaled somewhere, right? Can we just add json tags (in addition to the mapstructure tag) in types.go for marshaling so that the linter doesn't yell?

I need to confirm the marshalling, but if so then yeah that's a good suggestion to keep the code refs consistent. Let me look into it.

Reply:
Yep. 👍  
I think just the current change in eventResp's field should be enough.

Reply:
I don't think so because the 'EventId' in eventResp's field correspond to the objects in */types.go:

*EventId* definition is [here](https://github.com/aws/amazon-ec2-metadata-mock/blob/fd94034fd3e12fe60e44925a49cffe0c6c39f585/pkg/mock/scheduledevents/internal/types/types.go#L21) in scheduledevents/.../types.go

*md.EventId* is defined [here](https://github.com/aws/amazon-ec2-metadata-mock/blob/fd94034fd3e12fe60e44925a49cffe0c6c39f585/pkg/config/types.go#L119) in pkg/config/types.go

@bwagner5 , the multi-tag approach worked and the linter has been silenced 🤫

Updating this PR with revision